include_core/AtomicSupport.hpp

```diff
@@ -352,7 +353,11 @@ class VM_AtomicSupport
 #if defined(OMRZTPF)
 		cs((cs_t *)&oldValue, (cs_t *)address, (cs_t)newValue);
 		return oldValue;
-#elif defined(__xlC__) || defined(__open_xl__) /* defined(OMRZTPF) */
+#elif defined(J9ZOS390) /* defined(OMRZTPF) */
+        /* 390 cs() function defined in <stdlib.h>, doesn't expand properly to __cs1() which correctly deals with aliasing */
```

Comment:
as part of due diligence, we should also fix the formatting and grammar in all the code that was reordered. for example, ensuring that comments end with a full stop.